### DISCUSSION TITLE
UI Changes and Beginner Friendliness

### DISCUSSION THREAD

**Discussion by SpicyForthProgrammer** [Bithack/principia#29] [General]
The ui need improvement this is a discussion thread for that.

**Comment by SpicyForthProgrammer** [comment]
I will sort out parts later.

**Reply by SpicyForthProgrammer** [reply]
Done see pull 34

**Comment by SpicyForthProgrammer** [comment]
Desktop feels like playing a mobile game with mouse and keyboard and is very unintuitive

**Reply by sdac** [reply]
I agree with this, the game should be more PC-oriented and mobile should come second

**Reply by SpicyForthProgrammer** [reply]
I disagree i think it should be equal most of my time was spent on mobile. But mobile sandbox should be different from pc

**Reply by SpicyForthProgrammer** [reply]
They should be similar but little things would help. For instance moving camera with middle mouse button.

**Reply by rollerozxa** [reply]
> For instance moving camera with middle mouse button.

This! Some way of moving the camera without also moving junk around would be great.

**Reply by sdac** [reply]
Yeah that would be even better, two different modes

**Reply by rollerozxa** [reply]
About the sandbox menu picker, I personally don't know how it would be improved for desktop. I already tend to use the quickadd dialog almost exclusively and I assume most people would use that too if they've learnt the object selection and know what they're looking for.

**Reply by SpicyForthProgrammer** [reply]
The more objects that get added the harder it is to navigate sandbox menu. I will work on that later.

**Comment by SpicyForthProgrammer** [comment]
Sandbox object factory menu changes font size for every category.

**Reply by SpicyForthProgrammer** [reply]
How why.

**Reply by griffi-gh** [reply]
(it just takes the tallest item and stretches the rest to fit that size i think)

**Comment by griffi-gh** [comment]
I think the sidebar should use (vector) icon tab-based navigation instead of the current dropdown one

**Reply by griffi-gh** [reply]
Something like this:
![image](https://github.com/Bithack/principia/assets/45996170/167b265a-7a2e-499b-b6c6-2a37c4d158c6)

**Reply by griffi-gh** [reply]
hm removing recents hotbar is a bad idea...
what about this then:

![imgx](https://github.com/Bithack/principia/assets/45996170/1eec7fbe-d57d-46c4-8f1d-133ca65edd7f)

**Reply by griffi-gh** [reply]
hmm also what about a "Favorite items" menu and a category title at the top....
also paginated layout?
![imgx3](https://github.com/Bithack/principia/assets/45996170/73ccd594-af7c-4062-84af-d4730f210177)
